src/pages/JobVacancies/JobVacancies.tsx

```diff
@@ -68,17 +68,27 @@ export const JobVacancies: React.FC = (): React.ReactElement => {
     >
       <h1>Job Vacancies</h1>
       <div className="search-container">
-        <div className="search-bar">
-          <input
-            type="text"
-            id="candidateSearch"
-            name="candidateSearch"
-            placeholder="Enter your search"
-            value={search}
-            onChange={(e) => setSearch(e.target.value)}
-          />
-          <FontAwesomeIcon className="search-icon" icon={faSearch} />
+        <div className="search-bar-container">
+          <div className="search-bar">
+            <input
+              type="text"
+              id="candidateSearch"
+              name="candidateSearch"
+              placeholder="Enter your search"
+              value={search}
+              onChange={(e) => setSearch(e.target.value)}
+            />
+            <FontAwesomeIcon className="search-icon" icon={faSearch} />
+          </div>
+          <button
+            onClick={() => setOpenModal()}
+            className="standard-button add-vacancy"
```

Comment:
Might look good green?

--clr-button-primary: #33d188;

Reply:
Yes it does look better in green, I've updated it to suit